File: 프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.py
def solution(emergency):
    answer = []
    sorted_emergency = sorted(emergency, reverse=True)
    N = len(emergency)
    for i in range(N):
        for j in range(1, N+1):
            if emergency[i] == sorted_emergency[j-1]:
                answer.append(j)
    return answer

File: 프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.java
import java.util.Arrays;
class Solution {
    public int[] solution(int[] emergency) {
        int N = emergency.length;
        int[] answer = new int[N];
        int[] sorted = new int[N];
        for (int i = 0; i < N; i++) {
            sorted[i] = emergency[i];
        }
        Arrays.sort(sorted);
        for (int i = 0; i < N; i++) {
            for (int j = 0; j < N; j++) {
                if (emergency[i] == sorted[j]) {
                    answer[i] = N-j;
                    break;
                }
            }
        }
        return answer;
    }
}